packages/web/src/pages/settings-page/components/desktop/ManagerMode/RemoveManagerConfirmationContent.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import { useRemoveManager } from '@audius/common/api'
+import { Status } from '@audius/common/models'
+import { Button, Flex, Text } from '@audius/harmony'
+
+const messages = {
+  removeManagerConfirmation:
+    'Are you sure you want to remove this manager from your account?',
+  stopManagingConfirmation:
+    'Are you sure you want to stop managing this account?',
+  cancel: 'Cancel',
+  remove: 'Remove',
+  errorGeneral: 'Something went wrong.'
+}
+
+type RemoveManagerConfirmationContentProps = {
+  isStopManaging?: boolean
+  onSuccess: () => void
+  onCancel: () => void
+  userId: number
+  managerUserId: number
+}
+
+export const RemoveManagerConfirmationContent = ({
+  isStopManaging,
+  userId,
+  managerUserId,
+  onSuccess,
+  onCancel
+}: RemoveManagerConfirmationContentProps) => {
+  const [removeManager, result] = useRemoveManager()
+  const [error, setError] = useState<string | null>(null)
+  const { status } = result
+
+  const handleDelete = useCallback(() => {
+    if (!userId || !managerUserId) return
+    removeManager({ userId, managerUserId })
+  }, [userId, managerUserId, removeManager])
+
+  useEffect(() => {
+    if (status === Status.SUCCESS) {
+      onSuccess()
+    }
+  }, [status, onSuccess])
+
+  useEffect(() => {
+    if (status === Status.ERROR) {
+      setError(messages.errorGeneral)
+    }
+  }, [status])
+
+  if (!managerUserId) return null
+
+  const isSubmitting = status !== Status.IDLE
+
+  return (
+    <Flex direction='column' gap='xl'>
+      <Text variant='body' size='l'>
+        {isStopManaging
```

Comment:
nit: If this is only used to control the title, might be nice to just accept the title string/component as a prop instead of `isStopManaging` which is a little cryptic.

Reply:
yeah i went back and forth on this one. ill do that!